sass/_component_rst.scss

```diff
@@ -1,305 +0,0 @@
-.rst-content {
```

Comment:
I'm not sure how I feel about removing this. We don't currently use these styles, but they appear to be useful for styling the autodocs. See https://github.com/wagtail/sphinx_wagtail_theme/issues/131.

Let's just keep them for now, okay?